flexmeasures/data/schemas/sensors.py

```diff
@@ -55,14 +58,14 @@ class Meta:
 class SensorIdField(fields.Int, MarshmallowClickMixin):
     """Field that deserializes to a Sensor and serializes back to an integer."""
 
-    @with_appcontext
-    def _deserialize(self, value, attr, obj, **kwargs) -> Sensor:
+    @with_appcontext()
```

Comment:
Curious - what is the difference here? I believe we have many `@with_appcontext` usages without `()` in other places.

Reply:
Ah I see now. Maybe rename your custom one `with_appcontext_if_needed`?